### PR TITLE
Clearswift DLP Mapping

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5508)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ClearswiftDLP/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Clearswift DLP 
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Clearswift DLP to forward Syslog messages.
+
+Open your Clearswift instance UI and follow these instructions:
+1. From the Home page, click on **System** > Under Monitoring & Control, press on **Logs & Alarms**.
```
Comment (ShirleyDenkberg) [inline Packs/ClearswiftDLP/README.md:7]:
```suggestion
1. From the Home page, click **System** > **Monitoring & Control** > **Logs & Alarms**.
```

File: Packs/ClearswiftDLP/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+2. Click the **Log Export** tab and select **Enable log export**.
+3. Enter the IP Address (OR Hostname) and Port number of your Syslog server, the default port is 514.
```
Comment (ShirleyDenkberg) [inline Packs/ClearswiftDLP/README.md:9]:
```suggestion
3. Enter the IP Address (OR Hostname) and Port number of your Syslog server. The default port is 514.
```

File: Packs/ClearswiftDLP/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+4. Enter a Poll Interval (minutes). The recommended setting is once every 5 minutes, you can enter a value between 1 and 60 minutes.
```
Comment (ShirleyDenkberg) [inline Packs/ClearswiftDLP/README.md:10]:
```suggestion
4. Enter a Poll Interval (minutes). The recommended setting is once every 5 minutes. You can enter a value between 1 and 60 minutes.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.  However, I was not able to add the Docs Approved label. Can you help me with that?

**Comment by evisochek** [comment]
> @evisochek Doc review completed. However, I was not able to add the Docs Approved label. Can you help me with that?

Sure

**Review by evisochek — APPROVED** [review]

**Pull request merged**